step.go

```diff
@@ -1,19 +1,26 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
+	"strings"
+
 	"github.com/bitrise-io/go-steputils/stepconf"
 	"github.com/bitrise-io/go-utils/command"
 	"github.com/bitrise-io/go-utils/env"
 	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-steplib/bitrise-step-artifact-pull/model"
 )
 
 type Input struct {
-	Verbose bool `env:"verbose,required"`
+	Verbose         bool   `env:"verbose,required"`
```

Comment:
Is it only strange to me, that verbose is explicitly required? I know it is in the original step template, but why? Wouldn't the default false value be okay for the boolean var here?

Reply:
I think it is needed because of the input validation.

Reply:
Ah, I see.